### DISCUSSION TITLE
Open link in `:vsplit?`

### DISCUSSION THREAD

**Discussion by AndOrangutan** [nvim-neorg/neorg#334] [Q&A] [answered]
Hello,
I was wondering if anyone knows how to open a link in a `:vsplit`. Ideally, I would like to bind this to <S-Tab>.

**Comment by max397574** [comment]
<m-cr>

**Comment by mrossinek** [comment, marked as answer]
There is a default keybinding set to do exactly that. The default is `<M-CR>`.
If you have disabled the default keybindings you can specify it like so (warning this will change with #308):
```lua
local neorg_callbacks = require('neorg.callbacks')

neorg_callbacks.on_event("core.keybinds.events.enable_keybinds", function(_, keybinds)

keybinds.map_event_to_mode("norg", {
    n = {
        { "<M-CR>", "core.norg.esupports.hop.hop-link", "vsplit" },
        },
    }, { silent = true, noremap = true })
```

**Reply by esquires** [reply]
Good idea! Any chance we can adda keybind to also open in a new tab?

**Reply by mrossinek** [reply]
Great suggestion! I just added support to do so by specifying `"tab"` as the extra argument (i.e. instead of `"vsplit"` above).
There is no default mapping for this so you'll need to configure this one yourself :+1:

**Reply by esquires** [reply]
Thanks! Can a check be added for whether the link is already open in another tab/window? This would prevent multiple tabs with the same content from being opened when a link is followed multiple times.

**Reply by AndOrangutan** [reply]
Thanks :D
